src/app/components/VideoPlayer/index.js

```diff
@@ -105,13 +106,20 @@ const VideoPlayer = ({
     isMuted = true;
   }
 
-  if (!title) {
-    title = String.fromCharCode(getNextUntitledMediaCharCode());
+  if (typeof title === 'string') {
+    title = title.trim();
+  } else {
+    title = '';
   }
 
   const placeholderEl = Sizer(ratios);
 
-  const videoEl = html`<video preload="none" tabindex="-1" aria-label="${title}"></video>`;
+  const videoEl = html`<video
+    preload="none"
+    tabindex="-1"
+    aria-label="${title || ''}"
```

Comment:
We've already sanitised this above

```suggestion
    aria-label="${title}"
```